interfaces/policy/policy_test.go

```diff
@@ -790,6 +798,10 @@ plugs:
           - plugs-name-bound-p1
         slot-names:
           - plugs-name-bound-s2
+  attr-match-pub:
+    allow-auto-connection:
+      plug-attributes:
+        attr: $PLUG_PUBLISHER_ID
```

Comment:
what happens if we use the PLUG_PUBLISHER_ID constraint in a slot allow-installations stanza?

Reply:
maybe we don't have test for the same issue with PLUG and SLOT but maybe we should add them if not?

Reply:
It's not allowed: https://github.com/snapcore/snapd/blob/5655616d284b6ee058638f4a1a39facf34649b3d/interfaces/policy/helpers.go#L283
same thing for the slot side. I've added a test for it